assets/js/lib/locations/mealsLocation.js

```diff
@@ -0,0 +1,38 @@
+class MealsLocation {
+    constructor(dataRow) {
+      this.type      = "MealsLocation";
+      this.title     = dataRow[0];
+      this.latitude  = parseFloat(dataRow[2]);
+      this.longitude = parseFloat(dataRow[3]);
+  
+      this.contacts = [];
+  
+      if(dataRow[4]!== undefined) {
+        var contacts_info = dataRow[4].split(',')
```

Comment:
A suggestion: use camel case for variables to be consistent. `contactsInfo`. 

Also, prefer `let` instead of `var`.

Not critical but simple suggestions.